### ISSUE TITLE
Divisão do  Oradores Expediente

### ISSUE THREAD

**Issue by puera** [interlegis/sapl#3006]
## Comportamento Esperado
Em algumas casas legislativas , o expediente é dividido em Pequeno e Grande Expediente, seria interessante ter uma opção na tabelas auxiliares para poder ativar essa mudança, isso iria refletir apenas na lista dos Oradores do Expediente, nesse caso Ficaria 2 models para Oradores Pequeno Expediente e Oradores Grande Expediente.

## Comportamento Atual
O Expediente é único, não tendo possibilidade de adicionar oradores na divisão. 

## Seu Ambiente

* Versão usada (_Release_): 3.1.159

**Comment by edwardoliveira** [comment]
Sim, estamos cientes dessa limitação. Foi das primeiras coisas que notei ao mexer no SAPL. 

Mas, **FYI**, infelizmente, com nosso backlog de issues pendentes, isso provavelmente será feito só daqui uns anos somente. Mexer na estrutura de tabelas no Django é algo não trivial, sujeito a falhas e muito arriscado.
